strawberryfields/apps/similarity.py

```diff
@@ -47,50 +47,77 @@
 
 A feature vector of a graph can be created by choosing a collection of orbits or events and
 evaluating their probabilities with respect to GBS with the embedded graph. These
-probabilities are then selected to be elements of the feature vector. Evaluating the
-probabilities of orbits or events can be achieved through two approaches:
+probabilities are then selected to be elements of the feature vector. For example, event

```

Comment:
Note, can also update line 1 of this file to read:
`Copyright 2019-2020 Xanadu Quantum Technologies Inc.`